libp2p/protocols/secure/secio.nim

```diff
@@ -324,7 +325,7 @@ method handshake*(s: Secio, conn: Connection, initiator: bool = false): Future[S
     raise newException(SecioError, "Remote proposal decoding failed")
 
   if not remotePubkey.init(remoteBytesPubkey):
-    trace "Remote public key incorrect or corrupted", pubkey = remoteBytesPubkey
+    trace "Remote public key incorrect or corrupted", pubkey = remoteBytesPubkey.shortHexDump
```

Comment:
`shortHexDump` here is not useful here because in such case you will lose public key type information.